pkg/commands/process/settings/rules.go

```diff
@@ -18,18 +18,30 @@ var (
 )
 
 func RefreshRules(config Config, externalRuleDirs []string, options flag.RuleOptions, foundLanguages []string) (err error) {
-	builtInRules, rules, _, err := loadRules(externalRuleDirs, options, foundLanguages, true)
+	builtInRules, rules, _, bearerRulesVersion, err := loadRules(externalRuleDirs, options, foundLanguages, true)
```

Comment:
`loadRules` is now returning a lot of things; what's the best practice here?

Reply:
It might be better to return a struct for the non-error values.